src/auditor/cli.py

```diff
@@ -34,10 +37,22 @@ def cli():
         print(__doc__)
     else:
 
-        report_dir = args['--save_report']
-
-        org_auditor = Auditor(args['--verbose'])
-        org_auditor.check_items(report_dir)
-
-        if not args['--dry']:
-            org_auditor.fix_items(report_dir)
+        # report_dir = args['--save_report']
+
+        cli_logger = logging.getLogger('auditor')
+        cli_logger.setLevel(logging.DEBUG)
+        detailed_formatter = logging.Formatter(
+            fmt='%(levelname)-7s %(asctime)s %(module)10s:%(lineno)5s %(message)s', datefmt='%m-%d %H:%M:%S'
+        )
+        cli_handler = logging.StreamHandler(stream=sys.stdout)
```

Comment:
**thought**: if you are using stdout, then you shouldn't need any print statements. You can replace all your if verbose statements by setting the logging level.

Reply:
My understanding of the conversation on Slack was that forklift logs at the debug level and we didn't want 1300+ lines cluttering the forklift logs, so I left them as prints. It could be possible to define a custom log level, but that seems like just as much of a work around. Thoughts?

Reply:
It seems that you aren't using the standard forklift logger so wouldn't that be a non issue?

Reply:
I think I see the confusion, correct me if I'm wrong. You pass a logger object into `auditor` at instantiation, so when it's called as part of a pallet it uses forklift's logger, and when called from the CLI you pass in your own logger.

It also creates a custom logger for the results csv regardless of where it's instantiated, but this is only used for the results csv. All other log calls go to the logger passed in at instantiation.

Reply:
ok yeah that sounds about right. sorry for the confusion